logstash-core/lib/logstash/pipeline.rb

```diff
@@ -144,7 +145,7 @@ def run
 
     start_workers
 
-    @logger.log("Pipeline #{@pipeline_id} started")
+    puts "Pipeline #{@pipeline_id} started"
```

Comment:
As a logstash user, I would appreciate to have this log at `info` level for example.